docs/conf.py

```diff
@@ -21,7 +21,7 @@
 author = 'Gustav Pettersson @ ESA'
```

Comment:
is there a notion of multiple authors? I'd want to add my name, but not drop Gustav.

Reply:
i'll have to look at the docs for Sphinx, probably...